src/views/TasksDetailsView.vue

```diff
@@ -89,6 +89,46 @@ interface TaskStatusButton {
   action?: Function;
 }
 window.scrollTo({ top: 0, behavior: "smooth" });
+
+const pointsTemplateHeight = ref(80);
+const showPointsText = ref(true);
+
+onMounted(() => {
+  const checkHeight = () => {
+    const mobileLayout = document.querySelector(".mobile-layout") as HTMLElement;
+
+    if (mobileLayout) {
+      const layoutHeight = mobileLayout.offsetHeight;
+
+      if (layoutHeight > 179) {
+        pointsTemplateHeight.value = 124;
+        showPointsText.value = false;
+      } else {
+        pointsTemplateHeight.value = 80;
+        showPointsText.value = true;
+      }
+    }
```

Comment:
Dobrą praktyką jest nie zagnieżdzanie ze sobą ifów. Trzeba pisac kod w ten sposob by był on jak najbardziej płaski. Dodatkowo jeśli chcesz uzyc if-else na koncu funkcji to juz nie musisz pisac else'a, wystarczy ze dasz w ifie returna. Przyklad jak to powinno wygladac:

```suggestion
    if (!mobileLayout) {
      return;
    }

    const layoutHeight = mobileLayout.offsetHeight;
    if (layoutHeight > 179) {
      pointsTemplateHeight.value = 124;
      showPointsText.value = false;
      return;
    }
    
    pointsTemplateHeight.value = 80;
    showPointsText.value = true;
```

A najlepiej w tym konkretnym przypadku uzyc ternary operatora:
```suggestion
    if (!mobileLayout) {
      return;
    }

    const layoutHeight = mobileLayout.offsetHeight;
    pointsTemplateHeight.value = layoutHeight > 179 ? 124 : 80;
    showPointsText.value = layoutHeight > 179 ? false : true;
```

Obczaj: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator